core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -90,6 +90,7 @@ type TransmitChecker interface {
 // - existence of a saved eth_tx_attempt
 type EthBroadcaster struct {
 	logger    logger.Logger
+	orm       ORM
```

Comment:
broadcaster is for next PR, but I had to introduce orm here so the broadcaster could use `LoadEthTxAttempts` from the orm